corehq/apps/linked_domain/applications.py

```diff
@@ -107,16 +108,12 @@ def link_app_via_app_manager(master_domain, master_id, target_domain, target_nam
     DEPRECATED: this workflow is in the process of being phased out (hence the weird name)
     """
     linked_app = _create_linked_app(target_domain, target_name, master_id)
-    return link_app(linked_app, master_domain, master_id, remote_details)
+    return link_app(linked_app, master_domain, remote_details)
 
 
-def link_app(linked_app, master_domain, master_id, remote_details=None):
+def link_app(linked_app, master_domain, remote_details=None):
     DomainLink.link_domains(linked_app.domain, master_domain, remote_details)
 
-    linked_app.family_id = master_id
```

Comment:
I don't think it's a good idea to remove this while there's still code that uses `family_id`.